aten/src/ATen/core/boxing/KernelFunction.cpp

```diff
@@ -18,6 +19,16 @@ void fallthrough_kernel(OperatorKernel*, const OperatorHandle&, Stack*) {
     "let us know in the bug tracker.");
 }
 
+void named_not_supported_kernel(OperatorKernel*, const OperatorHandle& op, Stack*) {
+  // DO NOT LOOK AT STACK, YOU HAVE SHORT CIRCUITED BOXING
+  // See Note [named_not_supported_kernel]
+  TORCH_CHECK(0,
```

Comment:
Just curious, why 0 instead of false?

Reply:
No good reason. I probably was copying something nearby.